graph_auth/schema.py

```diff
@@ -9,22 +9,30 @@
 from django.db import models
 import django.contrib.auth
 from django.contrib.auth.forms import PasswordResetForm, SetPasswordForm
-from rest_framework_jwt.settings import api_settings
 from django.utils.http import urlsafe_base64_decode as uid_decoder
 from django.utils.encoding import force_text
+from mail_templated import EmailMessage
 
+from rest_framework_jwt.settings import api_settings
 from graph_auth.settings import graph_auth_settings
 
+UserModel = django.contrib.auth.get_user_model()
+
+class DynamicUsernameMeta(type):
+    def __new__(mcs, classname, bases, dictionary):
+        dictionary[UserModel.USERNAME_FIELD] = graphene.String(required=True)
```

Comment:
I don't think this needs to always be required. Or perhaps it might conflict with the explicit "email" field below.